### PR TITLE
Adds components for pypi

### PR DESCRIPTION
**Pull request by cdo03c**
Some slight changes to allow pushing to pypi.

Also, removed two dependencies: m2r2 and torch.  Tested that pymasq vignettes run correctly.

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -4,7 +4,7 @@
 [metadata]
 name = pymasq
 version = attr: pymasq.__version__
-license = MIT
+license = LICENSE.rst
```
Comment (howardgershon) [inline setup.cfg]: should this be LICENSE.txt

File: README.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ DISTRIBUTION STATEMENT A. Approved for public release. Distribution is unlimited
 
 © 2021 Massachusetts Institute of Technology.
 
-    Subject to FAR 52.227-11 – Patent Rights – Ownership by the Contractor (May 2014)
+Subject to FAR 52.227-11 – Patent Rights – Ownership by the Contractor (May 2014)
     SPDX-License-Identifier: Insert SPDX ID
```
Comment (howardgershon) [inline README.md]: We should replace "Insert SPDX ID" with "MIT" on line 57

Comment (howardgershon) [reply README.md]: Changed this

File: setup.cfg (hunk 1 above)
Comment (howardgershon) [reply setup.cfg]: Changed this

**Review by howardgershon — APPROVED** [review]

File: setup.cfg (hunk 1 above)
Comment (cdo03c) [reply setup.cfg]: done

File: README.md (hunk 2 above)
Comment (cdo03c) [reply README.md]: done

**Pull request merged**